mockgcp/common/operations/operations.go

```diff
@@ -60,6 +63,57 @@ func (s *Operations) NewLRO(ctx context.Context) (*pb.Operation, error) {
 	return op, nil
 }
 
+func (s *Operations) StartLRO(ctx context.Context, callback func() (proto.Message, error)) (*pb.Operation, error) {
+	now := time.Now()
+	millis := now.UnixMilli()
+	id := uuid.NewUUID()
+
+	op := &pb.Operation{}
+
+	op.Name = fmt.Sprintf("operations/operation-%d-%s", millis, id)
```

Comment:
YYYY-MM-DDTHH:mm:ss.sssZ maintains time sort order but has the advantage of being human readable. (Sorry I can't convert epoch millis to something more readable in my head) Might be worth using this. You can always use '-' if the ':' are a problem.

Reply:
Good point.  We're actually going to clean this up in a few commits, because we want to look as close to GCP as possible (so we can do golden testing of requests/responses)